LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2023 Diariatou Diakite
+
+Permission is hereby granted, free of charge, to any person obtaining a copy
+of this software and associated documentation files (the "Software"), to deal
+in the Software without restriction, including without limitation the rights
+to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+copies of the Software, and to permit persons to whom the Software is
```

Comment:
- [ ] It would be better if we create the MIT license file in our project and customize it accordingly.Yes you created the license file but you have not customise it accordingly to fit the Microverse template .please kindly fix it  
- [ ] your license should look like this
![Screenshot (116)](https://github.com/diarisdiakite/bookstore/assets/92458236/f84abfbe-ca82-46ed-8fef-5c1e81895cfe)

- [ ] Your lincense
![Screenshot (117)](https://github.com/diarisdiakite/bookstore/assets/92458236/cc514c9d-7b75-4bba-b4f4-d63f7a22ed98)